iris/static/js/lecturer_session.js

```diff
@@ -1,38 +1,42 @@
 const button_start = document.getElementById("button_start");
 const button_stop = document.getElementById("button_stop");
-const feedbackmsg = document.getElementById("lecturer_message")
+const feedbackmsg = document.getElementById("lecturer_message");
 let socket = io();
 socket.emit('join', {'course_id': courseID});
 
-if(sessionActive == "True"){
+function disableStart(){
     button_start.disabled = true;
     button_stop.disabled = false;
-}else{
+    feedbackmsg.innerHTML = "Session active";
+}
+
+function disableStop(){
     button_start.disabled = false;
     button_stop.disabled = true;
+    feedbackmsg.innerHTML = "Session not active";
+}
+
+if(sessionActive == "True"){
```

Comment:
sessionActive is set in the HTML, by the template. It is passed as a python boolean, and therefore is True. It could be possible to fix this in the template or in the backend. Will look into it.